src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (int i = 0; i < users.length; i++) {
+            if (users[i].getEmail().equals(email)) {
+                return users[i];
+            }
+        }
```

Comment:
Please rewrite with for-each loop, it will increase code readability

for(User user : users) {...